### PR TITLE
fix kotlin warnings in azure-destination, datastore-{bigquery,mongo,postgres} CDK submodules

### PR DESCRIPTION
**Pull request by stephane-airbyte**
cleaning kotlin warnings in some CDK submodules

### CODE REVIEW

**Comment by stephane-airbyte** [comment]
* **#37479** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/37479?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @stephane-airbyte and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: airbyte-cdk/java/airbyte-cdk/datastore-bigquery/src/main/kotlin/io/airbyte/cdk/db/bigquery/BigQueryDatabase.kt [hunk 1]
```diff
@@ -93,22 +93,20 @@ constructor(
...
-                    }
-                    .collect(Collectors.toList())
+            Arrays.stream(params)
+                .map { param: String? ->
+                    QueryParameterValue.newBuilder()
+                        .setValue(param)
+                        .setType(StandardSQLTypeName.STRING)
+                        .build()
+                }
+                .collect(Collectors.toList())
```
Comment (postamar) [inline airbyte-cdk/java/airbyte-cdk/datastore-bigquery/src/main/kotlin/io/airbyte/cdk/db/bigquery/BigQueryDatabase.kt:109]: params.map { ... }.toList()

**Review by postamar — APPROVED** [review]

File: airbyte-cdk/java/airbyte-cdk/azure-destinations/src/main/kotlin/io/airbyte/cdk/integrations/destination/jdbc/copy/azure/AzureBlobStorageStreamCopier.kt [hunk 2]
```diff
@@ -47,16 +47,18 @@ abstract class AzureBlobStorageStreamCopier(
         )
     protected val azureStagingFiles: MutableSet<String> = HashSet()
 
-    @get:VisibleForTesting val tmpTableName: String = nameTransformer.getTmpTableName(streamName)
+    @Suppress("DEPRECATION")
+    @get:VisibleForTesting
```
[the remark this answers is not in the document]
Comment (stephane-airbyte) [reply airbyte-cdk/java/airbyte-cdk/azure-destinations/src/main/kotlin/io/airbyte/cdk/integrations/destination/jdbc/copy/azure/AzureBlobStorageStreamCopier.kt:51]: haven't looked at that. Was focusing on kotlin warnings

[the remark this answers is not in the document]
Comment (stephane-airbyte) [reply airbyte-cdk/java/airbyte-cdk/azure-destinations/src/main/kotlin/io/airbyte/cdk/integrations/destination/jdbc/copy/azure/AzureBlobStorageStreamCopier.kt:51]: those are all over the place. Will need another pass for that :(

**Pull request merged**